### DISCUSSION TITLE
why i can't sign in to vercell with my github?

### DISCUSSION THREAD

**Discussion by Talith4** [vercel/community#140] [Help] [answered]
I want to join with vercel but I cannot join or sign in or log in to vercel with my account github. It's like my account github is blocked. Do you have any answer with my problem?

**Comment by leerob** [comment]
Could you share more information?

- Can you sign in with email?
- Is there an error shown?

**Comment by Pieparker** [comment, marked as answer]
This conversation appears to have gone quiet. To keep discussions on topic and to prevent the resurrection of old topics, this discussion has been closed. If issues persist or a future reader stumbles upon this discussion and is exhibiting the same problem, please create a new discussion.
